lib/createWorkers.js

```diff
@@ -1,350 +1,19 @@
 'use strict';
 
-const EventEmitter = require('events');
-const _ = require('lodash');
-const co = require('co');
-const url = require('url');
-const amqplib = require('amqplib');
+const stakhanov = require('stakhanov');
 const logger = require('chpr-logger');
 
-const {
-  promisifyWithTimeout,
-  subscribeToConnectionEvents,
-  subscribeToChannelEvents
-} = require('./lib');
-const {
-  applyConfiguration,
-  applyOptions
-} = require('./schema/createWorkers.schema');
-
-const DEFAULT_EXCHANGE_TYPE = 'topic';
-
 /**
  * Create a worker instance based on configuration
  * @param {Array} handlers array of handlers to handle each message
- * @param {function} handlers.handle function to handle each message
```

Comment:
Why did you remove this doc?

Reply:
When implementing a worker I only want to look at the packages documentation I directly use, not the underlying packages. I would keep the doc here since we are going to update the code anyway if we bump a major version of our dependencies

Reply:
I agree that duplication is a pain, but the jsdoc will help the IDE to suggest useful insights.
(note to myself: I'll have to document the new options when bumping this...)